package.json

```diff
@@ -4,8 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "vitest",
```

Comment:
👌

Reply:
The head branch was actually not correctly set, so the diff you were seeing was from https://github.com/xataio/client-ts/pull/173 😆

Reply:
Whops, my bad. Forgot to change the head branch when creating the PR.